src/DnTable/__stories__/index.stories.jsx

```diff
@@ -32,25 +33,31 @@ export const Default = () => {
     return (
         <DnTable>
             <DnTable.Header>
-                <DnTable.HeaderCell>Date</DnTable.HeaderCell>
-                <DnTable.HeaderCell>Time</DnTable.HeaderCell>
-                <DnTable.HeaderCell>User</DnTable.HeaderCell>
-                <DnTable.HeaderCell>Role</DnTable.HeaderCell>
-                <DnTable.HeaderCell>Cause</DnTable.HeaderCell>
+                <DnTable.Row>
+                    <DnTable.HeaderCell>Date</DnTable.HeaderCell>
+                    <DnTable.HeaderCell>Time</DnTable.HeaderCell>
+                    <DnTable.HeaderCell>User</DnTable.HeaderCell>
+                    <DnTable.HeaderCell>Role</DnTable.HeaderCell>
+                    <DnTable.HeaderCell>Cause</DnTable.HeaderCell>
+                </DnTable.Row>
             </DnTable.Header>
             <DnTable.Body>
-                <DnTable.HeaderCell>04/14/2017</DnTable.HeaderCell>
-                <DnTable.BodyCell>19:34 PDT</DnTable.BodyCell>
-                <DnTable.BodyCell>yby.jupiter</DnTable.BodyCell>
-                <DnTable.BodyCell>Admin</DnTable.BodyCell>
-                <DnTable.BodyCell>jira123</DnTable.BodyCell>
+                <DnTable.Row>
+                    <DnTable.Cell>04/14/2017</DnTable.Cell>
+                    <DnTable.Cell>19:34 PDT</DnTable.Cell>
+                    <DnTable.Cell>yby.jupiter</DnTable.Cell>
+                    <DnTable.Cell>Admin</DnTable.Cell>
+                    <DnTable.Cell>jira123</DnTable.Cell>
+                </DnTable.Row>
             </DnTable.Body>
             <DnTable.Footer>
-                <DnTable.HeaderCell>04/14/2017</DnTable.HeaderCell>
-                <DnTable.BodyCell>19:34 PDT</DnTable.BodyCell>
-                <DnTable.BodyCell>yby.jupiter</DnTable.BodyCell>
-                <DnTable.BodyCell>Admin</DnTable.BodyCell>
-                <DnTable.BodyCell>jira123</DnTable.BodyCell>
+                <DnTable.Row>
+                    <DnTable.Cell>04/14/2017</DnTable.Cell>
```

Comment:
can u please use `HeaderCell` as the previous example.
```suggestion
                    <DnTable.HeaderCell>04/14/2017</DnTable.HeaderCell>
```

Reply:
@shadiabuhilal are you referring to the `denali/css` storybook?

Reply:
My thinking was since the `frozen` styles are not applied in the simple example, we didn't need to use `HeaderCell`.

Reply:
Ok